_pages/plugins/gut-analysis-toolbox.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Gut Analysis Toolbox
+description: Gut Analysis Toolbox or GAT allows the semi-automated analysis of the cells within the enteric nervous system of the gastrointestinal tract in **2D**. GAT enables quantification of enteric neurons and their subtypes in gut wholemounts. It runs in FIJI, a popular image analysis software in microscopy and uses deep learning models to segment cells of interest. 
+categories: [Analysis,Segmentation,Neuron,Gut]
+---
+
+**Gut Analysis Toolbox** enables semi-automated analysis of the neuronal and glial distribution within the gut wall (enteric nervous system). It uses a combination of `FIJI macros`, [StarDist](https://github.com/stardist/stardist) 
+and [deepImageJ](https://deepimagej.github.io/deepimagej/) under the hood to do the heavy lifting with automated segmentation. 
+
+<p align="center">
+<img src="https://github.com/pr4deepr/GutAnalysisToolbox/blob/main/wiki_images/figures/summary_figure.png" alt="GAT_overview" width="600" >
```

Comment:
Does the `{% include img align="center" width=600 src="..." %}` not work here? See [the documentation](https://imagej.net/editing/images#alignment). Also, you probably meant https://raw.githubusercontent.com/pr4deepr/GutAnalysisToolbox/main/wiki_images/figures/summary_figure.png for the raw image, right? Note that you can commit images to this repository in the `/media/plugins` folder, if you would prefer to host it here, but either way is fine.